lib/scss/components/news-list-item/_list-item.scss

```diff
@@ -77,11 +108,63 @@
     @include modular-spacing('margin-bottom', 1);
 
     display: inline;
+    float: left;
+
+    @include grid-media-min('lg') {
+      max-width: unset;
+    }
+
+    @include grid-media-min('xl') {
+      max-width: 54%;
+    }
+
+    @include grid-media-min('2xl') {
+      max-width: 64%;
+    }
   }
 
   .su-news-list__dek {
     @include big-paragraph;
-    @include modular-spacing('margin-bottom', 2);
+    @include modular-spacing('margin-bottom', 1);
+    @include modular-spacing('margin-top', -1);
+
+    display: block;
+    clear: left;
+  }
+
+  ul {
```

Comment:
Why is there list styles in this?  The list item template no longer has any `ul` or `li` elements in it.

Reply:
This is the taxonomy list at the end of the article. Should it live elsewhere?

Reply:
well it doesn't have any selector for it and the element doesn't exist in the template. This is an difficult one because ideally the scss files should only target the classes and elements that are known to exist in the accompanied template file. Styles that add based on a view configuration are more difficult to decide where they live.